peertube/Makefile

```diff
@@ -33,10 +33,12 @@ config-hook:
 		echo && echo "Postfix configuration:" && \
 		${BIN}/reconfigure_ask ${ENV_FILE} PEERTUBE_POSTFIX_myhostname "Enter the domain of your email address" && \
 		# I use `dotenv set` in the next line instead of `reconfigure` because the value needs to have an "=" in it but `reconfigure` strips all "=" \
-		${BIN}/dotenv -f ${ENV_FILE} set PEERTUBE_OPENDKIM_DOMAINS="$$(${BIN}/dotenv -f ${ENV_FILE} get PEERTUBE_POSTFIX_myhostname)=postfix" && echo "Set PEERTUBE_OPENDKIM_DOMAINS=$$(${BIN}/dotenv -f ${ENV_FILE} get PEERTUBE_POSTFIX_myhostname)=postfix" && \
+		${BIN}/dotenv -f ${ENV_FILE} set PEERTUBE_OPENDKIM_DOMAINS="$$(${BIN}/dotenv -f ${ENV_FILE} get PEERTUBE_POSTFIX_myhostname)=postfix" && echo "Set PEERTUBE_OPENDKIM_DOMAINS=$$(${BIN}/dotenv -f ${ENV_FILE} get PEERTUBE_POSTFIX_MYHOSTNAME)=postfix" && \
```

Comment:
I still don't understand why this variable has the word "my" in it.

Reply:
I started from their example [`docker-compose.yml`](https://github.com/Chocobozzz/PeerTube/blob/master/support/docker/production/docker-compose.yml) and [`.env`](https://github.com/Chocobozzz/PeerTube/blob/master/support/docker/production/.env), and their `.env` has [`POSTFIX_myhostname`](https://github.com/Chocobozzz/PeerTube/blob/3d4d49a23eae71f3ce62cbbd7d93f07336a106b7/support/docker/production/.env#L43) in it.

My assumption was that the postfix container would be looking for an env var called exactly "POSTFIX_myhostname", so I didn't change it.

Reply:
I can't tell either. It looks like you should change the var name, but how would the postfix container know to get the value of `POSTFIX_BANANA`?

Reply:
I guess its a postfix thing, https://unix.stackexchange.com/questions/58243/which-name-should-be-used-for-myhostname-in-postfixs-main-cf

Reply:
ok whatevs.